Packs/Core/Playbooks/playbook-SSO_Brute_Force.yml

```diff
@@ -0,0 +1,1051 @@
+            iscontext: true
+          right:
+            value: {}
+        - operator: isEqualString
+          left:
+            value:
+              simple: Core.RiskyUser.risk_level
+            iscontext: true
+          right:
+            value:
+              simple: HIGH
+          ignorecase: true
+        - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: DBotScore
+                filters:
+                - - operator: greaterThanOrEqual
+                    left:
+                      value:
+                        simple: DBotScore.Score
+                      iscontext: true
+                    right:
+                      value:
+                        simple: "1"
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: DBotScore.Type
+                      iscontext: true
+                    right:
+                      value:
+                        simple: ip
+                    ignorecase: true
+            iscontext: true
+          ignorecase: true
+        - operator: isEqualString
+          left:
+            value:
+              simple: IntervalAnalysis.IsPatternLikelyAutomated
+            iscontext: true
+          right:
+            value:
+              simple: "true"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1090,
+          "y": 930
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "22":
+    id: "22"
+    taskid: d766566f-458b-49b1-87dc-59bfc76ce5ca
+    type: regular
+    task:
+      id: d766566f-458b-49b1-87dc-59bfc76ce5ca
+      version: -1
+      name: Get user MFA configuration to assess their resilience
+      description: Returns all the enrolled factors for the specified user.
+      script: '|||okta-get-user-factors'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "16"
+    scriptarguments:
+      extend-context:
+        simple: MFAResult=
+      username:
+        complex:
+          root: Core.OriginalAlert.event
+          accessor: auth_identity
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 850,
+          "y": 620
+        }
+      }
+    note: true
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "24":
+    id: "24"
+    taskid: 44da91f4-abdd-4773-819e-fadd023d4614
+    type: collection
+    task:
+      id: 44da91f4-abdd-4773-819e-fadd023d4614
+      version: -1
+      name: Decide if to expire the user password
```

Comment:
That doesn't sound like correct English to me, did @ShirleyDenkberg reviewed it?

Would change that to:
Decide whether to expire the user's password

Reply:
Yes.  That is better.